test/unittests/t_download.cc

```diff
@@ -62,7 +62,7 @@ class T_Download : public FileSandbox {
   static const char tmp_path[];
 
   perf::Statistics statistics;
-  DownloadManager download_mgr;
+  DownloadManager *download_mgr;
```

Comment:
Why does it need to become a pointer?

Reply:
i darkly remember something with that there were some issues with `Clone()`..